composites/Plugin/Shared/components/Icon.js

```diff
@@ -1,35 +0,0 @@
-import React from "react";
```

Comment:
I don't think this file has to be removed, because it can still be used to render svg react components. We could still have more complex react components (that use more than just a path) that are just plain react components that are passed to this component.

Reply:
OK, however the Icon component had also a IconTest.js, which required the edit.svg file.

Reply:
And should we keep `svg-react-loader` then?

Reply:
No, in case we keep using `Icon.js`, we start using SVG's are react components and not as svg files:

```js
// React component
const EditIcon = ( props ) => {
    return(
        <svg { ...props } >{ "..." }</svg>
    )
}